modules/crypt.cpp

```diff
@@ -501,25 +504,36 @@ class CCryptMod : public CModule {
         }
     }
 
+    void OnNickPrefixCommand(const CString& sCommand) {
+        CString sPrefix = sCommand.Token(1, true);
+
+        if (sPrefix.empty()) {
+            PutModule("Nick Prefix: " + NickPrefix());
+        } else {
+            CString sStatusPrefix = GetUser()->GetStatusPrefix();
+            size_t sp = sStatusPrefix.size();
+            size_t np = sPrefix.size();
+            SetNV(NICK_PREFIX_KEY, sPrefix);
+            PutModule("Setting Nick Prefix to " + sPrefix);
+            if (sStatusPrefix.CaseCmp(sPrefix, std::min(sp, np)) == 0)
+                PutModule("WARNING: overlap with Status Prefix (" + sStatusPrefix + "), this Nick Prefix will not be used!");
```

Comment:
Are you talking to me or to the user who tried to set such prefix?
*I* know about that function. But the current behavior would be surprising, and the message which user sees is misleading.